Green-space/clean.js

```diff
@@ -0,0 +1,68 @@
+"use strict";
+const turf = require("@turf/turf");
+const multipolygon = require("turf-multipolygon");
```

Comment:
Inside `@turf/turf` you'll have `multiPolygon` as a function.

Reply:
I think I was getting "not a function" error. Must be something stupid like case-sensitivity...

Reply:
Oh it is case sensitive, which Text editor are you using? TurfJS works really well with VSCode which allows for IntelliSense (autocomplete helper).

**IntelliSense in Visual Studio Code**

![image](https://user-images.githubusercontent.com/550895/34282769-135537fe-e696-11e7-8e52-0c248a21cc83.png)

Reply:
I'm using atom. There is probably intellisense-like plugin. Will check it out.